wandb/sdk/launch/builder/build.py

```diff
@@ -473,14 +460,14 @@ def _get_docker_image_uri(name: Optional[str], work_dir: str, image_id: str) ->
     return name + version_string
 
 
-def _create_docker_build_ctx(
+def _create_build_ctx(
```

Comment:
so everything here is no longer docker speciifc right? this is general build stuff?

Reply:
This does include the dockerfile, I'll put docker back in the function name.